README.md

```diff
@@ -196,3 +173,44 @@ Note that the contents of those file will be overidden on each run.
 
 We generate a self-signed wildcard certificate to use for all the ingress
 controllers.
+
+## Development and testing
+
+To bootstrap a development environment, you need to install the
+runtime dependencies listed above, plus the development environment
+dependencies.
+
+### Running the tests
+
+This section explains how to run linters and the compliance test suites.
+
```

Comment:
This is ok, but we are not closing issue #89 "We should write instructions on how to handle all these conventions." with this. Later on we could add a "FORMATTING.md" file and add a reference here.
FORMATTING.md could be a simple table, one row for each file type checked (.sh. yaml, Vagrantfile). For each file we put a link to the formatting rules that we are enforcing.

Reply:
Ok, I'll add a table, so we can close the issue